main.js

```diff
@@ -141,4 +141,17 @@ window.onclick = function clickOut(event) {
     closeModal();
     clearModal();
   }
-};
\ No newline at end of file
+};
+
+// validate form
+const contactForm = document.querySelector('#contact-form');
+const email = document.getElementById('email');
+const messageError = document.getElementById('email-error');
+
+contactForm.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    messageError.innerText = 'Your message was not sent, you need to have the email address, all in small letters';
```

Comment:
- [ ] Great job so far 👍 a user cannot submit the form with an email containing an uppercase letter. Kindly make the error message appear near the submit button as per the requirements 💯 
- Requirements:
![Annotation 2022-04-14 131825](https://user-images.githubusercontent.com/85113567/163389431-557a67e5-9ef8-4045-b10b-ff6cedd37e03.jpg)
- Your design:
![Annotation 2022-04-14 131743](https://user-images.githubusercontent.com/85113567/163389480-36678313-7432-436b-a8a1-a5bb0483bdf3.jpg)